### DISCUSSION TITLE
ALM parallelization

### DISCUSSION THREAD

**Discussion by RicardoCecconello** [ttadano/alamode#211] [Q&A] [answered]
Dear Dr. Tadano,

Is it possible to run ALM with MPI parallelization using more than one node?

**Comment by ttadano** [comment, marked as answer]
Unfortunately, MPI parallelization is not available within ALM. Only OpenMP parallelization is implemented.

**Reply by RicardoCecconello** [reply]
Alright, thanks for the reply!
